### ISSUE TITLE
Add option to ignore https certificate validation

### ISSUE THREAD

**Issue by darkrole** [Rob--W/cors-anywhere#71]
id like to know on which part of the code do i need to add this "NODE_TLS_REJECT_UNAUTHORIZED=0"
because im trying to access the API and getting this error 

404 Not Found
Not found because of proxy error: Error: self signed certificate

**Comment by Rob--W** [comment]
Set an environment variable with that value. Via the command line (Windows: https://ss64.com/nt/set.html, Linux/macOS: https://ss64.com/bash/export.html).

Via Node.js: https://github.com/Rob--W/cors-anywhere/blob/97c6e53020da622b006842ef112bc721cac0c47d/test/test.js#L394

**Comment by tsjnsn** [comment]
try this:
```javascript
cors_proxy.createServer({
    ...
    httpProxyOptions: {
      secure: false
    }
})
```

**Comment by Rob--W** [comment]
This was fixed by https://github.com/Rob--W/cors-anywhere/commit/3c87a51664bc625535afcecc4e618009ff41d2ce when I fixed #341 (which is a duplicate of this bug).
